junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/EnabledIfEnvironmentVariableCondition.java

```diff
@@ -28,21 +24,24 @@
  * @since 5.1
  * @see EnabledIfEnvironmentVariable
  */
-class EnabledIfEnvironmentVariableCondition implements ExecutionCondition {
+class EnabledIfEnvironmentVariableCondition
+		extends AbstractRepeatableAnnotationCondition<EnabledIfEnvironmentVariable> {
+
+	private static final ConditionEvaluationResult ENABLED = ConditionEvaluationResult.enabled(
+		"No 'disabled' @EnabledIfEnvironmentVariable conditions encountered");
```

Comment:
I see what you meant here, but the `disabled @Enabled...` may be a bit confusing, no? Have no better idea though...

Reply:
Indeed. That can certainly be confusing.

The thing is... every `ExecutionCondition` either returns `enabled()` or `disabled()`, and we only care if `disabled()` is the result. 

I guess we could change it to something like `No @EnabledIfEnvironmentVariable conditions resulting in 'disabled' execution encountered`.

Is that better?

Reply:
Yes, I find it more understandable indeed!

Reply:
Done